FindTown/FindTownUI/Sources/FindTownUI/PopUp/ContentPopUpViewController.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  ContentPopUpViewController.swift
+//  
+//
+//  Created by 장선영 on 2022/12/28.
+//
+
+import UIKit
+
+/// 사용자 지정 콘텐츠View가 들어가는 팝업 VC
+open class ContentPopUpViewController: UIViewController {
+        
+    public var contentView: UIView = {
+        let view = UIView()
+        view.backgroundColor = FindTownColor.white.color
+        view.layer.cornerRadius = 16
+        view.translatesAutoresizingMaskIntoConstraints = false
+        return view
+    }()
+    
+    public var topLeftButton: UIButton = {
```

Comment:
```suggestion
    public var cancelButton: UIButton = {
```
여기에 무슨 버튼이 들어갈지 모르지만, 아마 x 버튼만 들어갈 거 같아서요...!!